app/models/katello/erratum.rb

```diff
@@ -110,6 +120,10 @@ def self.list_filenames_by_clauses(repo, clauses)
           where(statement).pluck(:filename)
     end
 
+    def modular?
+      ::Katello::Erratum.modular.where(:id => id).exists?
+    end
```

Comment:
why is this method needed?  what exactly is the intent?

Reply:
Good call. Removing it. I was planning to show if an errata has modules or not. But that can be done in another way if needed so removing it.